lib/RequestHandler.js

```diff
@@ -11,19 +11,15 @@ export async function RequestHandler({
   schemaObj = null,
 }) {
   let session = {};
```

Comment:
no need to declare vairables outside as there is no usage, use directly where used ,and prefer const when not reassiging